### PR TITLE
Fix french sentences

### PR DESCRIPTION
**Pull request by zecakeh**
This is based on PR #1139.
I took @Booteille 's commit, rebased it to master to resolve conflicts and added my own commit on top to:
 -  Fix other spelling/grammar errors
 -  Remove duplicates
 -  Cut long sentences
 -  Keep valid (but unusual) spelling according to the "rectifications orthographiques du français en 1990"
 -  Add missing punctuation
 -  Add capital letter at the start of sentences
 -  Convert numbers to words
 -  Remove parenthesises and slashes

@lissyx if you want to have a look at it before @mikehenrty merges it.
When this is merged, PR #1139 can be closed.

### CODE REVIEW

**Comment by lissyx** [comment]
Thanks, I'll have a look. I'd like also a look from you on https://github.com/mozilla/voice-web/pull/1243 :)

**Comment by lissyx** [comment]
@zecakeh I guess that's too late, but it might have been easier to review with one commit per scope of of change: one for spelling, one for grammar, one for dupes, etc.

File: server/data/fr/ofis_publik_ar_brezhoneg.txt [hunk 1]
```diff
@@ -1,1153 +1,1167 @@
...
+Tu trouves le temps long.
+Où vont-ils ?
+Où vont-elles ?
+Où allions-nous ?
+Où voulais-tu que nous allions ?
+Où voulais-tu que l’on aille ?
+En cinquième.
+Ta tête.
+Tes têtes.
+À quelle heure vient-il ?
```
Comment (lissyx) [inline server/data/fr/ofis_publik_ar_brezhoneg.txt]: All those `A` should be `À`

**Comment by zecakeh** [comment]
@lissyx Yeah sorry about that. When I realised that I had already done a lot of work.

**Pull request merged**